libraries/lib-preferences/Prefs.h

```diff
@@ -702,5 +702,6 @@ struct PREFERENCES_API PreferenceInitializer {
 extern PREFERENCES_API StickySetting<BoolSetting> DefaultUpdatesCheckingFlag;
 extern PREFERENCES_API StickySetting<BoolSetting> SendAnonymousUsageInfo;
 extern PREFERENCES_API StickySetting<StringSetting> InstanceId;
+extern PREFERENCES_API StickySetting<IntSetting> IntroOrderStart;
```

Comment:
I don't think it has to be `sticky` setting, we could safely reset it to the default on settings reset.

Reply:
We need `StickySetting` just for a handful of settings that should stay even if the user resets all settings to default.
For example, we do not want to reset the InstanceId or show the Usage Info agreement window again.